src/Asm89/Twig/Lint/Command/LintCommand.php

```diff
@@ -124,4 +140,11 @@ protected function getOutput(CliOutputInterface $output, $format)
                 throw new RuntimeException(sprintf("Unknown output format '%s'.", $format));
         }
     }
+
+    protected function printSummary(array $stats, OutputInterface $output)
```

Comment:
Add `summary()` to the `OutputInterface` and make the csv implementation a no op for now.